CreateDB

```diff
@@ -46,13 +49,13 @@ CREATE TABLE Likes(
     mid INTEGER REFERENCES Messages(mid),
     uid INTEGER REFERENCES Users(uid),
     upvote BOOLEAN NOT NULL,
```

Comment:
Not really a change we need to do now but something to keep in mind. We should probably consider renaming this table to something else. As long as the ER say's likes, even is it has a dislikes attribute, it is not straightforward or easily interpreted. Recall that the first thing the professor questioned was that he saw a likes **relation** but not a dislikes **relation**. Note that he spoke of relations, not attributes.

Reply:
El issue era que we didn’t mention the attribute en el ER. Se puede hacer de varias formas. We could have two many-to-many relations de likes y dislikes or we could have una tabla de “feelings” sea like or dislike separada por un atributo. We meant to have one single table con atributo, we just forgot to mention the attribute en el ER. We should keep the tablea as similar as possible to the ER. Which I think having one relation instead of two is better

Reply:
You pretty much described what I said so yeah. We an take care of that later